mne/viz/topo.py

```diff
@@ -819,8 +823,9 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
     vmax : float
         The max value in the image. The unit is uV for EEG channels,
         fT for magnetometers and fT/cm for gradiometers.
-    colorbar : bool
-        Display or not a colorbar.
+    colorbar : bool | None
+        Whether to display a colorbar or not. If ``None`` a colorbar will be
```

Comment:
Instead of `a colorbar` please use `the colorbar` because there is always only one.

Reply:
Also, since you introduced the quoting in the docstring (``None``), do it consistently everywhere.